compiler/aarch64/codegen/OMRMachine.cpp

```diff
@@ -124,11 +489,212 @@ TR::RealRegister *OMR::ARM64::Machine::assignOneRegister(TR::Instruction *curren
    return assignedRegister;
    }
 
+/* generate instruction for register copy */
+static void registerCopy(TR::Instruction *precedingInstruction,
+                         TR_RegisterKinds rk,
+                         TR::RealRegister *targetReg,
+                         TR::RealRegister *sourceReg,
+                         TR::CodeGenerator *cg)
+   {
+   TR::Node *node = precedingInstruction->getNode();
+   TR::RealRegister *zeroReg;
+   switch (rk)
+      {
+      case TR_GPR:
+         zeroReg = cg->machine()->getARM64RealRegister(TR::RealRegister::xzr);
+         generateTrg1Src2Instruction(cg, TR::InstOpCode::orrx, node, targetReg, zeroReg, sourceReg, precedingInstruction); /* mov (register) */
+         break;
+      case TR_FPR:
+         TR_ASSERT(false, "Not implemented yet.");
```

Comment:
Is there a reason why this is left unimplemented?  This function is certainly called for FPRs.

Reply:
I am not able to implement this yet because `FMOV (register)` instruction is not available in OMRInstOpCodeEnum.hpp and OpBinary.cpp.  Other floating-point instructions, such as `FADD` and `FNEG`, are also missing.